idea/src/org/jetbrains/kotlin/idea/intentions/ConvertBlockCommentToLineCommentIntention.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.psi.PsiComment
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiWhiteSpace
+import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.psi.KtPsiFactory
+
+class ConvertBlockCommentToLineCommentIntention : SelfTargetingIntention<PsiComment>(
+    PsiComment::class.java, "Replace with end of line comment"
+) {
+    override fun isApplicableTo(element: PsiComment, caretOffset: Int): Boolean {
+        return element.isBlockComment()
+    }
+
+    override fun applyTo(element: PsiComment, editor: Editor?) {
+        val psiFactory = KtPsiFactory(element)
+
+        val prevSibling = element.prevSibling
+        val indent = if (prevSibling is PsiWhiteSpace) {
+            val space = prevSibling.text.reversed().takeWhile { it == ' ' }
```

Comment:
What about `'\t'`? Of course it's rare case but it's possible

Reply:
Please check how this and other intention work with situation like below (multiple new-lines between comment and declaration).

```
/* 
   block comment 
*/


fun foo() {}
```